container/container_unix.go

```diff
@@ -168,6 +169,21 @@ func (container *Container) SecretMountPath() string {
 	return filepath.Join(container.Root, "secrets")
 }
 
+// ConfigsDirPath returns the path to the directory where configs are stored on
+// disk.
+func (container *Container) ConfigsDirPath() string {
+	return filepath.Join(container.Root, "configs")
+}
+
+// ConfigFilePath returns the path to the on-disk location of a config.
+func (container *Container) ConfigFilePath(configRefTarget *swarmtypes.ConfigReferenceFileTarget) (string, error) {
```

Comment:
This is referring to the path on the host. It gets bind-mounted at the target path inside the container.